swap/swap.go

```diff
@@ -80,8 +85,41 @@ func NewParams() *Params {
 	}
 }
 
+// newLogger returns a new logger
+func newLogger(logpath string) log.Logger {
+	swapLogger := log.New("swaplog", "*")
+
+	defaultHandler := log.Root().GetHandler()
+	rfh, err := swapRotatingLogHandler(logpath)
+
+	if err != nil {
+		//default handler just in case the handler is not set
```

Comment:
this comment needs a verb in order to be clearer. are we _setting_ the default handler? if so, say so, and place the comment on the line that actually does this.

and when we say `the handler`, which one are we talking about?

Reply:
fixed